__tests__/app.test.js

```diff
@@ -5,98 +5,139 @@ const seed = require('../db/seeds/seed.js')
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(String),
+          })
         });
-    })
+      });
+  })
 })
 
 describe('3. GET /api/articles/:article_id', () => {
-    test('status:200, responds with a single matching article', () => {
-      const articleId = 3;
-      return request(app)
-        .get(`/api/articles/${articleId}`)
-        .expect(200)
-        .then((response) => {
-            const article = response.body.article
-          expect(article).toEqual({
-            author: "icellusedkars",
-            title: 'Eight pug gifs that remind me of mitch',
-            article_id: articleId,
-            body: "some gifs",
-            topic: 'mitch',
-            created_at: "2020-11-03T09:12:00.000Z",
-            votes: 0,
-          });
+  test('status:200, responds with a single matching article', () => {
+    const articleId = 3;
+    return request(app)
+      .get(`/api/articles/${articleId}`)
+      .expect(200)
+      .then((response) => {
+        const article = response.body.article
+        expect(article).toEqual({
+          author: "icellusedkars",
+          title: 'Eight pug gifs that remind me of mitch',
+          article_id: articleId,
+          body: "some gifs",
+          topic: 'mitch',
+          created_at: "2020-11-03T09:12:00.000Z",
+          votes: 0,
         });
-    });
-    test('400: invalid article_id', () => {
-        return request(app)
-        .get('/api/articles/bananas')
-        .expect(400)
-        .then((response) => {
-            const msg = response.body.msg;
-            expect(msg).toBe('Bad Request')
-        })
-    })
+      });
+  });
+  test('400: invalid article_id', () => {
+    return request(app)
+      .get('/api/articles/bananas')
+      .expect(400)
+      .then((response) => {
+        const msg = response.body.msg;
+        expect(msg).toBe('Bad Request')
+      })
+  })
+
+  test('404: valid article_id but doesnt exist', () => {
+    return request(app)
+      .get('/api/articles/30')
+      .expect(404)
+      .then((response) => {
+        const msg = response.body.msg;
+        expect(msg).toBe('Not Found')
+      })
+  })
+
+});
 
-    test('404: valid article_id but doesnt exist', () => {
-        return request(app)
-        .get('/api/articles/30')
-        .expect(404)
-        .then((response) => {
-            const msg = response.body.msg;
-            expect(msg).toBe('Not Found')
+describe('4. GET /api/articles/:article_id/comments', () => {
+  test('status:200, responds with comments from a single matching article_id', () => {
+    const articleId = 3;
+    return request(app)
+      .get(`/api/articles/${articleId}/comments`)
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body
+        expect(comments).toBeInstanceOf(Array);
+        comments.forEach((comment) => {
```

Comment:
It would be good to add an assertion to check the array length is what we'd expect to prevent false positives if we'd expect comments but the array is empty